tanner/data/psql_setup.py

```diff
@@ -0,0 +1,6 @@
+from sqlalchemy import create_engine
+engine=create_engine("postgresql://postgres:<pwd>@localhost/postgres")
+conn=engine.connect()
+conn.execute("commit")
+conn.execute("create database tanner_db")
```

Comment:
on tanner startup, run `CREATE TABLE IF NOT EXISTS tanner` (also rename the db to just `tanner`)

Reply:
what should i add for columns?

Reply:
what should i add for columns?

Reply:
you mean creating the tables?
We want to replicate the same functionality we have with Redis. Have a look at the session_manager and dorks.